src/zenml/environment.py

```diff
@@ -254,6 +256,20 @@ def in_github_codespaces() -> bool:
             or "GITHUB_CODESPACES_PORT_FORWARDING_DOMAIN" in os.environ
         )
 
+    @staticmethod
+    def in_zenml_codespace() -> bool:
+        """If the current Python process is running in ZenML Codespaces.
+
+        Returns:
+            `True` if the current Python process is running in ZenML Codespaces,
+            `False` otherwise.
+        """
+        return (
```

Comment:
Can't we simply decide on 1? 😄

Reply:
@schustmi which one do you like ? I like `ZENML_ENVIRONMENT=codespace`